packages/docusaurus-init/templates/bootstrap/docusaurus.config.js

```diff
@@ -54,5 +54,14 @@ module.exports = {
       copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
     },
   },
-  presets: [['@docusaurus/preset-bootstrap', {}]],
+  presets: [
+    [
+      '@docusaurus/preset-bootstrap',
+      {
+        blog: {
```

Comment:
Options are not needed if we stick with the standard names in the classic theme.